packages/Geographic/README.md

```diff
@@ -0,0 +1,51 @@
+## iTowns Geographic
```

Comment:
Just a thought: we may update the doc template to be structured per subpackage and export this page as the welcome page of the subpackage. WDYT @gchoqueux @Desplandis?

Reply:
yes, I think it would be recommended for future implementations to structure documentation by sub-module.